x-pack/plugins/ml/public/plugin.ts

```diff
@@ -26,6 +27,7 @@ export interface MlSetupDependencies {
   licensing: LicensingPluginSetup;
   management: ManagementSetup;
   usageCollection: UsageCollectionSetup;
+  licenseManagement: LicenseManagementUIPluginSetup;
```

Comment:
Here the same

```js
licenseManagement?: LicenseManagementUIPluginSetup;
```

Reply:
Changes in 73e832b65c118e802e2e5bb1a85d834bee95f5af